docs/conf.py

```diff
@@ -3,28 +3,28 @@
 # CvDisinfo-Detect documentation build configuration file, created by
 # sphinx-quickstart.
 #
-# This file is execfile()d with the current directory set to its containing dir.
+# This file is execfile() with the current directory set to its containing dir.
 #
 # Note that not all possible configuration values are present in this
 # autogenerated file.
 #
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-import os
-import sys
+# import os (original imported these, however flake8 said they were unused)
+# import sys (original imported these, however flake8 said they were unused)
 
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 # sys.path.insert(0, os.path.abspath('.'))
```

Comment:
Find a better way to deal with paths than this. I recommend always using *relative imports* in your project for imports to your own code. If you want to treat them as absolute paths instead, you need to properly pip install your project as a package.